### PR TITLE
improve contributor docs

### PR DESCRIPTION
**Pull request by BenTheElder**
- ungroup tool installs, most everyone should install git... generally improve the getting started guide
- add many links and a few more details to the scope document

### CODE REVIEW

**Comment by BenTheElder** [comment]
these docs both still need work. this should help a little.

**Comment by tao12345666333** [comment]
I also happened to think of this part, I pushed an [image](https://hub.docker.com/r/taobeier/kindenv/tags) last night. 

The source code https://github.com/tao12345666333/kindenv .

Do we need to provide a standard development environment, like the moby project's development environment?

**Comment by BenTheElder** [comment]
> Do we need to provide a standard development environment, like the moby project's development environment?

I don't think that's necessary currently. We just require a reasonably up to date Go install and the same docker install requirements as **using** kind. 
Git probably isn't worth containerizing just for kind, and Hugo is fairly optional since we have the previews and the contents are just markdown.

**Comment by tao12345666333** [comment]
I agree with you. :+1: 

The original intention of this [project](https://github.com/tao12345666333/kindenv) was to add a timing build task for CI. Auto build node image synchronized with the upstream version of kubernetes

**Comment by BenTheElder** [comment]
Ooh, related to https://github.com/kubernetes-sigs/kind/issues ? 👀

**Comment by tao12345666333** [comment]
xref: https://github.com/kubernetes-sigs/kind/issues/197

File: site/content/docs/contributing/getting-started.md [hunk 1]
```diff
@@ -20,44 +20,53 @@ an account to contribute.
...
-#### Install git
-Install `git` on your local machine.
 You can check if `git` is already on your system and properly installed with 
 the following command:
 
 ```
 git --version
 ```
-This documentation is written using `git` version 2.17.1. 
+This documentation is written using `git` version `2.17.1`.
```
Comment (neolit123) [inline site/content/docs/contributing/getting-started.md]: written -> managed?

also possibly no need to add this sentence?
users without git will land with a fairly new and compatible git package as a download.

Comment (BenTheElder) [reply site/content/docs/contributing/getting-started.md]: SGTM, this was already in the doc, I only changed the command / version markup to match. Will remove.

Comment (BenTheElder) [reply site/content/docs/contributing/getting-started.md]: done

**Pull request merged**